packages/ui/src/components/va-affix/VaAffix.vue

```diff
@@ -18,6 +18,7 @@
 <script lang="ts">
 import { defineComponent, computed, PropType, ref, Ref, nextTick, onMounted, onBeforeUnmount } from 'vue'
```

Comment:
This issue is resolved in the `dev` mode but still exists in the `production` mode.

Steps to reproduce:

1. Run yarn serve:production
2. Press "Should set dark red color on click" button in the `VaConfig`



> `VaConfig` -> `Global config` works while using dev variables (`yarn serve`) But does not work in the production mode (`yarn serve:production`) same behavior we can expect after the build.
> 
> Steps to reproduce:
> 
> 1. Press "Should set dark red color on click" button in the `VaConfig`
> 2. "Must change" should be red after this in production
> 
> Production mode: ![VaConfigProductionMode](https://user-images.githubusercontent.com/7745899/162422206-13deee99-f1c8-4d7a-9184-24075a245dc4.png)
> 
> Expected: ![VaConfigExpected](https://user-images.githubusercontent.com/7745899/162423036-7e1dac82-a6ec-4892-9c5b-21928348bfa2.png)

Reply:
Update:
I've noticed that "Must change color" is our `va-chip` `component`, for `va-button` it works correctly.

Tested the same thing on the build version of the project and it works the same.

Reply:
I think this happens because `va-chip` uses `VueClassComponent` inside unlike `va-button` and this case is not handled in the new logic - we have fixed similar cases in the previous version of `WithConfigTransport`.

Reply:
So we wait until VaChip will be refactored. I don't want to patch vue-class-component instance... It's made of getters 😭

Reply:
I can do this but looks like Oleg started it already.

Reply:
Fixed in https://github.com/epicmaxco/vuestic-ui/pull/1668

I rebased this branch with current develop. Works well in `yarn serve:production` and `yarn build && serve ./dist`.